crates/router/src/connector/adyen/transformers.rs

```diff
@@ -1057,6 +1100,44 @@ impl<'a> TryFrom<(&types::PaymentsAuthorizeRouterData, &api::Card)> for AdyenPay
     }
 }
 
+impl<'a> TryFrom<(&types::PaymentsAuthorizeRouterData, &api_models::payments::BankDebitData)> for AdyenPaymentRequest<'a>{
+    type Error = Error;
+
+    fn try_from(value: (&types::PaymentsAuthorizeRouterData, &api_models::payments::BankDebitData)
+    ) -> Result<Self, Self::Error> {
+        let (item, bank_debit_data) = value;
+        let amount = get_amount_data(item);
+        let auth_type = AdyenAuthType::try_from(&item.connector_auth_type)?;
+        let shopper_interaction = AdyenShopperInteraction::from(item);
+        let recurring_processing_model = get_recurring_processing_model(item);
+        let browser_info = get_browser_info(item);
+        let additional_data = get_additional_data(item);
```

Comment:
Does Adyen support manual capture for bank debits? Cause `get_additional_data` will add authorisation_type type in the request?